capnpc-go/capnpc-go.go

```diff
@@ -372,16 +371,27 @@ func (g *generator) Value(rel *node, t schema.Type, v schema.Value) (string, err
 		var buf bytes.Buffer
 
 		sd, err := g.data.copyData(data)
-		if err == nil {
-			switch t.AnyPointer().Which() {
-			case schema.Type_anyPointer_Which_unconstrained:
-				err = g.anyPointerUnconstrained(&buf, t, sd)
+		if err != nil {
+			return "", err
+		}
+
+		switch t.AnyPointer().Which() {
+		// Unconstrained?
+		case schema.Type_anyPointer_Which_unconstrained:
 
-			default:
-				err = g.pointerValue(&buf, sd)
+			switch t.AnyPointer().Unconstrained().Which() {
+			// Capability type?
+			case schema.Type_anyPointer_unconstrained_Which_capability:
+				return "nil", nil
```

Comment:
This could probably use a comment explaining why it's ok to assume the value is nil.

Reply:
Fixed.